scripts/integration_test_js.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Copyright (C) 2019 Zilliqa
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# This script will start an isolated server and run the python API against it
+#
+
+# Firstly determine if the user is running this locally, or is this in the CI
+
+RUNNING_LOCALLY=1
+
+if [[ $# -ne 1 ]];
+then
+    echo 'Assuming user is running on own system' >&2
+else
+    case $1 in
+        "--setup-env"|"--SETUP-ENV")  # Ok
+            RUNNING_LOCALLY=0
+            ;;
+        *)
+            # The wrong first argument.
+            echo '--setup-env or nothing' >&2
+    esac
+fi
+
+if [[ "$RUNNING_LOCALLY" == 1 ]]; then
+    echo "You are running the script locally. This is just for CI to run and you can do it yourself by running the test in tests/EvmAcceptanceTests against isolated server."
+    echo ""
+else
+    echo "The CI is running this script."
+    # Install dependencies silently on the CI server
+
+    echo "NOT starting isolated server - it should be running already"
+    ps -e | grep isolated
+
+    # install dependencies
+    apt update
+    apt -y install curl dirmngr apt-transport-https lsb-release ca-certificates
+    curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
didn't you have any issue with 14? maybe those failing tests were related to this. can you please test with 16?